pwa/src/components/PatientInfoForm.tsx

```diff
@@ -0,0 +1,265 @@
+  const [albendazole, setAlbendazole] = useState(0); // Default: No
+  const [visit, setVisit] = useState(0); // Default: No
+  const [pregnant, setPregnant] = useState(0); // Default: No
+  const [planning, setPlanning] = useState(0); // Default: No
+
+  // Data for Radio Buttons
+  const sexValues = [
+    { value: 0, label: "Male" },
+    { value: 1, label: "Female" },
+  ];
+
+  const radioYesNo = [
+    { value: 0, label: "No" },
+    { value: 1, label: "Yes" },
+  ];
+
+  /* istanbul ignore next */
+  const onSubmit = () => {
+    // This shows how to access the values of the form
+    const firstNameField: HTMLInputElement = document.getElementById(
+      "firstname"
+    ) as HTMLInputElement;
+    const lastNameField: HTMLInputElement = document.getElementById(
+      "lastname"
+    ) as HTMLInputElement;
+    const nicknameField: HTMLInputElement = document.getElementById(
+      "nickname"
+    ) as HTMLInputElement;
+    const birthdateField: HTMLInputElement = document.getElementById(
+      "birthdate"
+    ) as HTMLInputElement;
+    const complaintField1: HTMLInputElement = document.getElementById(
+      "complaint1"
+    ) as HTMLInputElement;
+    const complaintField2: HTMLInputElement = document.getElementById(
+      "complaint2"
+    ) as HTMLInputElement;
+    const complaintField3: HTMLInputElement = document.getElementById(
+      "complaint3"
+    ) as HTMLInputElement;
+
+    console.log("First name:", firstNameField.value);
+    console.log("Last name:", lastNameField.value);
+    console.log("Nickname:", nicknameField.value);
+    console.log("Birth Date:", birthdateField.value);
+    console.log(
+      "Complaints: ",
+      complaintField1.value +
+        " " +
+        complaintField2.value +
+        " " +
+        complaintField3.value
+    );
+    console.log("Wants Planning?: " + planning);
+  };
+
+  return (
+    <div id="patient-info-wrapper">
+      <TextareaItem
+        title="First Name"
+        placeholder="First Name"
+        autoHeight
+        id="firstname"
+      />
+
+      <TextareaItem
+        title="Last Name"
+        placeholder="Last Name"
+        id="lastname"
+        autoHeight
+      />
+
+      <TextareaItem
+        title="Nickname"
+        placeholder="Nickname"
+        id="nickname"
+        autoHeight
+      />
+
+      <List renderHeader={() => "Sex"}>
+        {sexValues.map((i) => (
+          <RadioItem
+            data-testid={`${i.label.toLowerCase()}-button`}
+            id={`${i.label.toLowerCase()}-button`}
+            key={i.value}
+            name="sex"
+            checked={sex === i.value}
+            onClick={() => setSex(i.value)}
+          >
+            {i.label}
+          </RadioItem>
+        ))}
+      </List>
+
+      <WhiteSpace />
+
+      <InputItem placeholder="YYYY-MM-DD" id="birthdate">
+        Birth Date
+      </InputItem>
+
+      <InputItem type="number" placeholder="Age" id="age">
```

Comment:
Limit to 3 character input maybe? Unless we have a breakthrough in human lifespan soon.

Or this field could be auto-calculated based on the date of birth, which would save us from having to validate this field.

Reply:
I like the idea of being auto-calculated by DOB, and I had considered that both fields are likely not necessary. However it was noted that patients may not know their exact date of birth & the two fields will not always be consistent with each other, so I left both in.

Reply:
[This commit](https://github.com/Haiti-Distributed-Healthcare-System/hdhs/pull/66/commits/c7baf1f1c2d3ea4474336e737ba432a0f3e78341) limited it to three characters for now.

I've created an issue label `form-optimizations` with disc "this is related to optimizing data entry". Auto-Calculation by DOB is now here: #73

Reply:
resolving this

Reply:
Looks Good.